sycl/doc/extensions/ExplicitSIMD/dpcpp-explicit-simd.md

```diff
@@ -0,0 +1,731 @@
+simd vector with the size determined by the template **__EltTy** parameter.
+- ```format<__EltTy, __Height, __Width>( )```: returns a reference to the calling simd object interpreted
+as a new 2D simd_view object with the shape determined by the template parameters **__Height** and**__Width**. The size of the new 2D block must not exceed the size of the original object.
+
+```cpp
+  simd<int, 16> v1;
+   // ...
+  auto v2 = v1.format<short>;
+  // v2 is a reference to the location of v1
+  // interpreted as a vector of 32 shorts.
+  // ...
+  auto m1 = v1.format<int, 4, 4>;
+  // m1 is a reference to the location of v1
+  // interpreted as a matrix 4x4 of ints.
+  // ...
+  auto m2 = v1.format<char, 4, 16>( );
+  // m2 is a reference to the location of v1
+  // interpreted as a matrix 4x16 of chars.
+```
+
+To model predicated move, Explicit SIMD provides the following merge functions:
+- ```merge(value_type __Val, __mask_type __Mask)```: this merge operation takes one source operand **__Val** and a mask **__Mask** defined as unsigned short vector of the same length. The semantic is that if the LSB of an element of **__Mask** is set, then the corresponding data element of **__Val** is copied to the corresponding position in the calling simd object.
+
+```cpp
+  simd<int, 4>   m, src;
+  simd<unsigned short, 4> mask;
+  // ...
+  m.merge(src, mask);
+  // m           src         mask      --->  m
+  // 2 2 2 2     4 4 4 4     1 1 0 1         4 4 2 4
+```
+
+- ```merge(value_type __Val1, value_type __Val2, __mask_type __Mask)```: this merge operation takes two source operands **__Val1** and **__Val2** as well as a simd mask. The semantic is that if the LSB of an element of **__Mask** is set, then the corresponding data element of **__Val1** is copied to the corresponding position in the calling simd object. Otherwise the corresponding data element of **__Val2** is copied to the corresponding position in the calling simd object.
+
+```cpp
+  simd<int,4>   m, src1, src2;
+  simd<unsigned short, 4> mask;
+  // ...
+  m.merge(src1, src2, mask);
+  // m           src1        src2         mask        --->  m
+  // 2 2 2 2     4 4 4 4     3 3 3 3      1 1 0 1           4 4 3 4
+```
+### `simd_view` class
+
+The ```sycl::intel::gpu::simd_view``` represents a "window" into existing simd object,
+through which a part of the original object can be read or modified. This is a
+syntactic convenience feature to reduce verbosity when accessing sub-regions of
+simd objects. **__RegionTy** describes the window shape and can be 1D or 2D,
+**__BaseTy** is the original simd object type, which can be a ```simd_view```
+itself.
+
+```simd_view``` allows to model hierarchical �views� of the parent ```simd```
+object�s parts, read/modify its elements through the views. Views can be of
+different shapes and dimensions as illustrated below (`auto` resolves to a
+`simd_view` instantiation):
```

Comment:
Lots of weird formatting bugs in this paragraph (e.g. views)

Reply:
thanks. copy-paste from PowerPoint